### ISSUE TITLE
utterances signin looks a bit ominous

### ISSUE THREAD

**Issue by andylolz** [devinit/D-Portal#580]
Specifically, this permission:
![Screenshot 2020-06-30 at 21 52 58](https://user-images.githubusercontent.com/464193/86175933-12f8e000-bb1c-11ea-8f3a-1f1c844a0c56.png)

[More details here](https://github.com/utterance/utterances/issues/23#issuecomment-405825267) and [here](https://github.com/MicrosoftDocs/visualstudio-docs/issues/619#issuecomment-368942234), I think… I wonder if github maybe reworded this permission, and it now sounds more worrying than before.

Anyway, just flagging because it would stop me from signing in. (Also, suspect github-only oauth is a blocker for many).

**Comment by xriss** [comment]
Yeah I don't think it can require anything less as it does create an issue via your github account and all permissions are scary. But it is open source, only uses the barest of servers because it has to, etc etc. It's not exactly tiktok.

Posting an issue is all it does, so the sort of people that would not want to use it because their github account is precious, well, they can already do that, no need for them to use it.

We are just seeing if we can prompt any non github people to sign in and post comments/issues here. I suppose you could say, this is a test to see if people would keep going to the point of creating a github account from that link on d-portal.

If you know of anything better.... :)

**Comment by andylolz** [comment]
Gotcha, I see. (FWIW I had no idea from the prompt that it was going to create an issue on github. I thought it was just using github for auth, and the comment would be stored somewhere else as per getsatisfaction or similar.)

If the goal is to get new users to create github issues, then this seems like the best thing to me.

**Comment by notshi** [comment]
Thanks, @andylolz! Really appreciate your comments - I've now added a note to explicitly mention that it will create an issue.

**Comment by andylolz** [comment]
That’s cool – thank you.

I should say as a general point: it is great that you are innovating on this. Thanks both!
